src/objects/InstancedMesh.js

```diff
@@ -193,6 +202,56 @@ class InstancedMesh extends Mesh {
 
 	}
 
+	setMorphAt( index, source ) {
+
+		if ( this.instanceMorph === null ) {
+
+			this.instanceMorph = new InstancedBufferAttribute( new Float32Array( this.instanceMatrix.count * 4 ), 4 );
+
+		}
+
+		if ( source.isVector4 ) {
+
+			source.toArray( this.instanceMorph.array, index * 4 );
+
+		} else if ( source.isMesh && source.morphTargetInfluences ) {
+
+			let index1 = - 1;
+			let index2 = - 1;
+
+			const objectInfluences = source.morphTargetInfluences;
+
+			for ( let i = 0; i < objectInfluences.length; i ++ ) {
+
+				const v = objectInfluences[ i ];
+
+				if ( v ) {
+
+					if ( index1 < 0 ) {
+
+						index1 = i;
+
+					} else if ( index2 < 0 ) {
+
+						index2 = i;
+
+					}
+
+				}
+
+				//morphInfluencesSum += v;
+
+			}
+
+			this.instanceMorph.array[ index * 4 ] = index1;
+			this.instanceMorph.array[ index * 4 + 1 ] = objectInfluences[ index1 ];
+			this.instanceMorph.array[ index * 4 + 2 ] = index2;
+			this.instanceMorph.array[ index * 4 + 3 ] = objectInfluences[ index2 ];
```

Comment:
So this means only two morph targets can influence a vertex at the same time, right? Maybe we should add this as a small comment so it becomes immediately clear what's going on in this code section.

Reply:
Yes it only supports simple animations, we could potentially support everything by using a texture to store all the infuences for the instances but it will be somwewhat slower and the PR will become more complex so I'm not sure there's a point in doing that as it is already quite useful like that. I'll add a comment and think what other improvements/cleanup I can do

Reply:
I'm struggling a bit with supporting just two influences. I fear we could quickly end up in a situation where this is reported as a bug and devs request full support. 

How would a solution with an additional texture look like?

Reply:
We would load all the influences in a float texture, like a row per instance with NUM_INFLUENCES floats and then fetch all of them in the shader using gl_InstanceId. Obviously this would be way slower than the current approach and how much slower would depend on the number of morph targets.  It will also add indirection, ie coordinates of texture fetches depending on previous texture fetches, that used to be a big deal but I'm not so sure with modern architectures. 

In the case of simple animations like the horse all but two of the influences will be zero so it's a big waste of bandwidth and from my experience this kind of morphing is the overwhelming majority of cases which would benefit from instancing. Stuff like animated pointers and little visual details that repeat a lot in the scene and just loop. 

We could always support both ways and allow the user to select with a flag. What do you think?